src/pear/config.vsh.yaml

```
@@ -0,0 +1,145 @@
+functionality:
+  name: pear
+  description: "PEAR is an ultrafast, memory-efficient and highly accurate pair-end read merger"
```

add more info from homepage

```suggestion
  description: |
    PEAR is an ultrafast, memory-efficient and highly accurate pair-end read merger. It is fully parallelized and can run with as low as just a few kilobytes of memory.
    
    PEAR evaluates all possible paired-end read overlaps and without requiring the target fragment size as input. In addition, it implements a statistical test for minimizing false-positive results. Together with a highly optimized implementation, it can merge millions of paired end reads within a couple of minutes on a standard desktop computer.